_layouts/tms.html

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+parent: TMS
+---
+
+<h1>
+  {{ page.title }}
+</h1>
+<h2>
+  {{ page.description }}
+</h2>
+
+<hr/>
+
+<p>
+  {% if page.api_integrations.size == 1 %}
+    {% if page.type contains "tms" %}
```

Comment:
I think up above, we just assign a variable.
```
{% if  page.type contains "tms" %}
  {% assign product_type =  "translation management system (TMS)" %}
{% else %}
  {% assign product_type =  "computer-assisted translation (CAT) tool" %}
{% endif %}
```

But even better would be a variable in generate.py, like you had.

```
> The <strong>{{ page.title }}</strong> {{ page.tms_type_name }} has <a href="#api-support">{{ page.api_integrations | size }} machine translation API integrations</a>.
```